sources/PTYTextView.m

```diff
@@ -1434,7 +1435,8 @@ - (void)keyDown:(NSEvent*)event {
         // track the instance of PTYTextView that is currently handling a key event and rerouting
         // calls as needed in -insertText and -doCommandBySelector.
         gCurrentKeyEventTextView = [[self retain] autorelease];
-        [self interpretKeyEvents:[NSArray arrayWithObject:event]];
+        _keydownEvent = event;
+        _inputMethodIsInserting = !event.isARepeat && [self.inputContext handleEvent:event];
```

Comment:
I think I see what's happening. Key repeat is funny in Mac OS these days. If you go to TextEdit and hold down <kbd>j</kbd> it does not repeat. If you hold down <kbd>n</kbd> it gives you a menu with `ñ` and `ń` to choose. So I guess it really is just key repeat that's special. It's nothing to do with `handleEvent:` vs `interpretKeyEvents:`, though.  The key difference is whether we call `[self.delegate keyDown:]` in `-[PTYTextView keyDown:]` since `insertText:replacementRange:` does not get called when there is key repeat.